### ISSUE TITLE
Handle app version names that do not follow the X.X.X pattern

### ISSUE THREAD

**Issue by stkent** [stkent/GooglePlayReviewScraper#1]
Examples to consider:

- `X.X.X.X`
- `X.X.X-XXX`

**Comment by macklinu** [comment]
Perhaps we could utilize a library like [k-bx/python-semver](https://github.com/k-bx/python-semver) to handle version comparisons.

**Comment by stkent** [comment]
Nice! We'll have to see how it handles some of the more unusual cases, but even if it only replaces the current (manual, bad) comparisons it'll be a good add.

**Comment by stkent** [comment]
I started working on this using `distutils.version` but then realized that the current code considers the app to have been updated just when the version string changes. Therefore, we do not currently care about more detailed comparisons at all. If in the future we decide to (1) save all historical data, AND (2) display said data, grouped by app version, in a chronological manner, THEN we might need to be able to handle proper comparisons. Closing the issue once you reply to confirm you've read this!
